research/global_moth_model/gradio/gradio_demo.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+import gradio as gr
+import PIL
+import torch
+from dotenv import load_dotenv
+from model_inference import ModelInference
+
+# Load secrets and config from optional .env file
+load_dotenv()
+GLOBAL_MODEL = os.getenv("GLOBAL_MODEL")
+CATEGORY_MAP = os.getenv("CATEGORY_MAP_JSON")
+CATEG_TO_NAME_MAP = os.getenv("CATEG_TO_NAME_MAP")
```

Comment:
It could be useful to also have default values when loading env variables, by adding the `default="Insert default value"`, so the code can still functions somewhat if there is no .env file (of course, this is not to be done for passwords and other sensible data)

Reply:
Done in [142ef3e](https://github.com/RolnickLab/ami-ml/pull/46/commits/142ef3e24251950af37f284b45c8a29fabee4665). I intially didn't understand this comment but now I do. I have added dummy variable names (like `global_model.pt` for the `GLOBAL_MODEL` variable) so that the code still runs without the `.env` file but throws an error for missing file.